shell/server/proxy.js

```diff
@@ -1549,6 +1597,10 @@ class Proxy {
         response.setHeader("Set-Cookie", rpcResponse.setCookies.map(makeSetCookieHeader));
       }
 
+      const mainUrl = process.env.ROOT_URL;
+      response.setHeader("Content-Security-Policy", "frame-ancestors " + mainUrl);
```

Comment:
Firefox is warning me:
```
Content Security Policy: Directive ‘frame-src’ has been deprecated. Please use directive ‘child-src’ instead.
```
which agrees with https://content-security-policy.com/